### ISSUE TITLE
Errors in description of the math constants

### ISSUE THREAD

**Issue by slichtzzz** [fontforge/fontforge#5385]
Looks like one of descriptions of the math constants lack correctness in the "fontforge/mathconstants.c"

https://github.com/fontforge/fontforge/blob/master/fontforge/mathconstants.c#L67
The description referred above reads: "Minimum distance between the baseline of an upper limit and the **bottom** of the base operator.", when it should be: "Minimum distance between the baseline of an upper limit and the **top** of the base operator."

In any way the same function is described on https://learn.microsoft.com/en-us/typography/opentype/spec/math
as "Minimum distance between baseline of upper limit and **(ink) top** of the base operator."

**Comment by iorsh** [comment]
Good catch, indeed the FF has wrong description there. Other similar constants look fine.
